yb-voyager/cmd/importData.go

```diff
@@ -358,16 +358,32 @@ func createVoyagerSchemaOnTarget(connPool *tgtdb.ConnectionPool) error {
 			PRIMARY KEY (schema_name, file_name)
 		);`,
 	}
-	for _, cmd := range cmds {
-		log.Infof("Executing on target: [%s]", cmd)
-		err := connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
-			_, err := conn.Exec(context.Background(), cmd)
-			return false, err
-		})
-		if err != nil {
-			return fmt.Errorf("create ybvoyager schema on target: %w", err)
+
+	maxAttempts := 12
+	currAttempt := 0
+	err := connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
```

Comment:
Do not use `connPool`. We used `newTargetConn()` to make it easier to implement retries.
```
outer:
for _, cmd := range cmds {
  for attempt := 1; attempt <= 12; attempt++ {
    log.Infof(...)
    if conn == nil {
      conn = newTargetConn()
    }
    _, err = conn.Exec(...)
    if err == nil {
      continue outer;
    }
    log.Warnf("Error while running [%s] attempt %d: %s", cmd, attempt, err)
    time.Sleep(5 * time.Second)
    conn.Close()
    conn = nil
  }
  if err != nil {
    utils.ErrExit("...")
  }
}
```

Reply:
👍